src/cryptonote_core/service_node_rules.cpp

```diff
@@ -280,4 +280,18 @@ bool get_portions_from_percent_str(std::string cut_str, uint64_t& portions) {
   return false;
```

Comment:
I'm pretty sure this function won't be used anymore (after deleting the pre-hf19 registration code) so we should nuke it.